dysql/pydantic_mappers.py

```diff
@@ -71,6 +72,17 @@ def _map_dict(self, current_dict: dict, record: sqlalchemy.engine.Row, field: st
         else:
             current_dict[model_field_name] = {record[field]: record[value_field]}
 
+    def _map_list_from_string(self, current_dict: dict, record: sqlalchemy.engine.Row, field: str):
+        list_string = record[field]
+        if self._has_been_mapped():
+            return
+
+        if list_string and isinstance(list_string, str):
+            values_from_string = list_string.split(',')
+
+            if field not in current_dict or not current_dict[field]:
+                current_dict[field] = values_from_string
```

Comment:
What happens if it does already exist? Should it extend it?

Reply:
originally I thought about doing it that way, and maybe thats what it should do ? in my case there should only be 1 record but maybe others would expect it to extend ?